README.md

```diff
@@ -99,4 +100,4 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 <!-- prettier-ignore-end -->
 <!-- ALL-CONTRIBUTORS-LIST:END -->
 
-This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind are welcome!
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind are welcome! Let's grow together!
```

Comment:
Remove one exclamation mark (!) from the ending of this line.

```... welcome ! Let's grow together!```

🚀 Add a Full stop (.) instead of Exclamation mark after text `welcome`